dipy/reconst/ivim.py

```diff
@@ -153,20 +166,18 @@ def nlls_fit(data, gtab, guess_params):
 
     Returns
     -------
-    nlls_params: the S0, f, D_star, D value for each voxel.
+    ivim_params: the S0, f, D_star, D value for each voxel.
 
     """
     flat_data = data.reshape((-1, data.shape[-1]))
     # Flatten for the iteration over voxels:
     bvals = gtab.bvals
     ivim_params = np.empty((flat_data.shape[0], 4))
     for vox in range(flat_data.shape[0]):
-        popt, pcov = curve_fit(ivim_function, bvals, flat_data[vox],
-                               guess_params[vox])
-        ivim_params[vox, :4] = popt
+        res = leastsq(_ivim_error,
```

Comment:
That's because minimize expects to minimize a function with a single output, the sum of squared errors (SSE). See here: https://github.com/sahmed95/dipy/pull/1